test/unit/src/common/components/repository-input/t_repository-input.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import expect from 'expect';
 import { shallow } from 'enzyme';
-import configureStore from 'redux-mock-store';
-
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
 // Workaround for Redux-wrapped components
 // See: https://github.com/airbnb/enzyme/issues/472
 import Component from '../../../../../../src/common/components/repository-input';
 const RepositoryInput = Component.WrappedComponent;
 
-let dispatch = expect.createSpy();
+//let dispatch = expect.createSpy();
```

Comment:
A fair bit of commented-out code in this file; I'd like to see that either explained or removed.